gcampus/core/templates/gcampuscore/components/trophic_index.html

```diff
@@ -0,0 +1,75 @@
+{% load i18n measurement %}
+<div class="bg-light me-2">
+    <div class="my-auto p-4">
+
+        <div class="row">
+            <div class="col-9">
+                <div class="text-secondary fw-bold pb-2">
+                    {% translate "Trophic Index" %}
+                </div>
+                {% if measurement.trophic_index.validity >= 0.25 %}
+                    <div class="index-classification display-1 fw-bolder trophic-class-{{ measurement.trophic_index.classification }}">
+                        {% translate "Class" %} {{ measurement.trophic_index.classification }}
+                    </div>
+                    <div class="index-classification display-4 fw-bolder bach-class-{{ measurement.trophic_index.classification }}">
+                        {{ measurement.trophic_index.description }}
+                    </div>
+                {% else %}
+                    <div class="index-classification display-1 fw-bolder class-unknown">
+                        {% translate "N/A" context "no value provided (water name, comment, etc)" as not_any %}
```

Comment:
Same problem as in `gcampus/core/templates/gcampuscore/components/bach_index.html`.